### PR TITLE
feat: All user manager working

### PR DESCRIPTION
**Pull request by lebojo**
User can connect, create account, change profile pictures and add friends!
Not security tested
Not paintested

### CODE REVIEW

File: README.md
Comment (JeyDot-dev) [inline README.md:1]: BRING BACK THE READMEEEEEEEE AAAAAAAAAAAARRRRHHH NOOOOOOO Q______Q

File: django-settings/dev-settings.py [hunk 1]
```diff
@@ -42,7 +42,6 @@
     "django.contrib.sessions",
     "django.contrib.messages",
     "django.contrib.staticfiles",
-    "database.apps.DatabaseConfig",
```
Comment (JeyDot-dev) [inline django-settings/dev-settings.py]: We need this D:

**Review by JeyDot-dev — APPROVED** [review]
Probablement peak. Je veux test 👯

**Review by Forgotw — APPROVED** [review]
User management intelligent et professionnel, tout en gardant une expérience utilisateur agréable.

**Pull request merged**